presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java

```diff
@@ -364,6 +364,110 @@ private Session enableDecimalParsing()
                 .build();
     }
 
+    @Test
+    public void testDecimalArithmetic() {
+        // Addition of two Long Decimal columns of type DECIMAL(35,20) and DECIMAL(29,4)
```

Comment:
I feel reviewers might be confused as to why the type is `DECIMAL(35,20)` when none of the inputs carry that. Should we simply say `decimal '999999999999999.999'` like below and clarify that the inferred column type is `DECIMAL(35,20)`